packages/layout/tests/layout.test.tsx

```diff
@@ -42,6 +43,44 @@ describe("<Badge />", () => {
   })
 })
 
+describe("<Container />", () => {
```

Comment:
I love seeing our test suite getting larger 💖

Reply:
As it say is the [Contribution Guidelines](https://github.com/Zyclotrop-j/chakra-ui/blob/develop/CONTRIBUTING.md#tests): 

> All commits that fix bugs or add features need a test.

😛 😉